packages/editor-ui/src/components/Sticky.vue

```diff
@@ -146,6 +147,18 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.$store.commit('setActiveNode', null);
 			}
 		},
+		onMarkdownLinkClick(event: Event) {
+			if(event.target && event.currentTarget) {
+				// Track clicks on sticky links
+				const clickedURL = (event.currentTarget as Element).getAttribute('href') || '';
+				// If it's a YT link and it contains a N8n quickstart thumbnail, it's a welcome video link
+				const isWelcomeVideo = clickedURL.startsWith('https://www.youtube.com/') &&
+				 ((event.target as Element).localName === 'img' && (event.target as Element).getAttribute('alt') === 'n8n quickstart video');
```

Comment:
why do we need to check for youtube as domain or event localName? should not `alt` be enough?

Reply:
I think not. I added it so it can be a bit of security that it's a video link but that won't work if video is hosted on other platforms. Agree about `alt` we just have to be careful to use that exact `alt` value for this telemetry to work.